taxi/views.py

```diff
@@ -64,14 +65,16 @@ class CarDetailView(LoginRequiredMixin, generic.DetailView):
 
 class CarCreateView(LoginRequiredMixin, generic.CreateView):
     model = Car
-    fields = "__all__"
+    # fields = "__all__"
```

Comment:
Please, remove this attribute here and in the other places as well where commented